Tests/TuistGeneratorTests/Settings/DefaultSettingsProviderTests.swift

```diff
@@ -773,7 +773,7 @@ final class DefaultSettingsProvider_iOSTests: TuistUnitTestCase {
 
         // Then
         XCTAssertSettings(got, containsAll: appTargetEssentialReleaseSettings)
-        XCTAssertEqual(got.count, 9)
+        XCTAssertEqual(got.count, 10)
```

Comment:
This is necessary because [this PR](https://github.com/tuist/XcodeProj/pull/920) introduced a new default setting (breaking change), and didn't catch the issue [in this PR](https://github.com/tuist/tuist/pull/7411) because I skipped Codemagic's CI because of the annoyance of not passing the GITHUB_TOKEN causing `mise install` to fail.